geometry_msgs/PoseStamped pick_pose
geometry_msgs/PoseStamped place_pose
---
bool success
string message